http/oidc/src/main/java/org/wildfly/security/http/oidc/ElytronMessages.java

```diff
@@ -234,5 +234,8 @@ interface ElytronMessages extends BasicLogger {
     @Message(id = 23056, value = "No message entity")
     IOException noMessageEntity();
 
+    @Message(id = 23057, value = "Attribute '%s' is not supported")
```

Comment:
s/Attribute/OIDC attribute

Reply:
fixed! and also making the same changes on the other RFE